plugin/parser/differ/mysql/differ.go

```diff
@@ -103,7 +104,49 @@ func buildTableMap(nodes []ast.StmtNode) map[string]*ast.CreateTableStmt {
 	return oldTableMap
 }
 
-// isTwoColumnsSame returns true if the two columns are the same.
-func isTwoColumnsSame(old, new *ast.ColumnDef) bool {
+// isTwoColumnsWithSameNameSame returns true if definitions of two columns with the same name are the same.
+func isTwoColumnsWithSameNameSame(old, new *ast.ColumnDef) bool {
```

Comment:
This name is a bit verbose. 
The `same column with the same name` equals `the same column`.
How about extra checking the name here and renaming it to `isSameColumns`?

Reply:
isColumnEqual sounds good, hope tidb parser to provide Equal()...

Reply:
Done.